app/routes/markets/index.js

```diff
@@ -79,4 +79,9 @@ router.get('/:marketKey', (req, res, next) => {
   });
 });
 
+router.post('/delete', (req, res, next) => {
+  db.deleteMarket(req.body.name);
+  res.send('deleting');
```

Comment:
If `db.deleteMarket` returns anything, send that back in the response.

Reply:
It doesn't return anything, I'll just have it return true.  It would only allow for 4 post requests to happen before stopping sending any.